src/core/server/integration_tests/saved_objects/routes/bulk_delete.test.ts

```diff
@@ -86,6 +86,7 @@ describe('POST /api/saved_objects/_bulk_delete', () => {
 
     const result = await supertest(httpSetup.server.listener)
       .post('/api/saved_objects/_bulk_delete')
+      .set('x-elastic-internal-origin', 'kibana')
```

Comment:
I went with adding the header rather than disabling the restriction because it reflects how end-users would make requests to these endpoints.